gridData/core.py

```diff
@@ -426,10 +426,13 @@ def export(self, filename, file_format=None, type=None):
 
         Parameters
         ----------
+
         filename : str
             name of the output file
+
         file_format : {'dx', 'pickle', None} (optional)
             output file format, the default is "dx"
+
         type : str (optional)
             for DX, set the output DX array type, e.g., "double" or
             "float"; note that PyMOL only understands "double" (see
```

Comment:
In the light of your comment 
> The most recent versions of PyMol now support type float: schrodinger/pymol-open-source@4bbe425
which is a reasonable decision, given that it is actually not parsing doubles correctly :-)

is this comment in the docs still relevant?

Reply:
It may still be relevant if the numbers in the DX map are outside the allowable range for a 32-bit float, regardless of which type is stated in the DX header.  You may want to actually *discourage* using `type=double` because Pymol doesn't really support it.

However, this is really a Pymol issue, and in all practical scenarios it is avoided now with the new format.  So I've just removed the comment for simplicity.